azure/storage-account-sample/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.yanchware.fractal</groupId>
+    <artifactId>storage-account.sample</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <name>storage-account-sample</name>
+    <url>https://github.com/YanchWare/fractal-samples</url>
+    <description>Sample of App service and related entities using Fractal Cloud SDK</description>
```

Comment:
Is the description correct? Or should it mention Storage Accounts as well?

Reply:
No it's not correct, I'll fix it.
Thank you!